### ISSUE TITLE
Hover text on events doesn't show

### ISSUE THREAD

**Issue by magnuskson** [j4321/tkcalendar#43]
Hi, 

When I put the cursor over an event in the calendar, I see no hovering message, nothing seems to happen. I have tested using the code in the example you have supplied, and that doesn't work. 
Windows 10, Python 3.7.4 and tkcalendar 1.4.

I can't find any other reports on this, so am I doing something wrong?

Thank you

**Comment by j4321** [comment]
Hello,

If you have just run the example named 'Calendar with events', then it should work (though the default delay before the message appearance is a bit long).  
Since it does not work for you, it is probably a bug. I do not use Windows but I am a bit surprised it does not work because tkcalendar's tests passed on Windows. 

I do not have a lot of time right now but I will try to run the example in Windows 10 as soon as possible to see if I can reproduce your issue.

**Comment by magnuskson** [comment]
Yes, I run "Calendar with events" and waited several seconds for the delay, without result.
The hover function is not essential to my application, but I thought I should report the problem.

**Comment by j4321** [comment]
Thanks for reporting, I will try to reproduce the issue.

**Comment by bjacobse** [comment]
Hi just to confirm I have same error using the example2, that hovering isn't working/showing
Windows 10, Python 3.7.4 and tkcalendar 1.5.1

**Comment by kqrsn** [comment]
Hi, I managed to get the tool tips to appear for "Calendar with events". I'm running Windows 10, Python 3.6.0, and 1.5.1. When the Calendar with events is initially opened up, the tool tips don't display. However after maximizing and restoring down the window, the tool tips began to display themselves properly.

**Comment by j4321** [comment]
@kqrsn Thanks for the information.

**Comment by alexsmith005** [comment]
I have the same issue. The tooltip doesn't initially show up, but after maximizing and minimizing, the tooltip populates. I've run the same code on a Mac and it works fine. Is there any work around for this aside from manually maximizing and minimizing? I tried to run the wm_state from normal to zoomed to normal and this doesn't work.

**Comment by iMampi** [comment]
Hi, I found a workaround.
I faced the same issue as the others said. the tooltip won't show up at first when u try unless you maximize or minimize the window (windows10, python 3.7,6 tkcalendar 1.5.1)
Here's what you have to do:
   In the example2, just add "top.update_idletasks()" directly below "top = tk.Toplevel(root)"
 So you'll have :

    def example2():

        top = tk.Toplevel(root)
        top.update_idletasks()
        ...

**Comment by teauxfu** [comment]
Can confirm the suggestion by @iMampi works
Windows 10, python 3.9, tkcal 1.5.0

Interestingly, the fix seems not to work if you call `update_idletasks` after initializing the `Calendar` 😕 
May well just be a `tkinter` issue

**Comment by mustangCAR** [comment]
Removing the `selectmode='none'` attribute from the Calendar definition fixes it for me...
